romancal/regtest/test_match_skycell.py

```diff
@@ -51,13 +51,13 @@ def test_match_files(run_patchlist, expected_match_files):
 
 
 @pytest.fixture(scope="module")
-def run_skycellasn(rtdata_module):
+def run_skycellasn(rtdata_module, run_patchlist):
```

Comment:
@ddavis-stsci , I somewhat blindly added this argument to run_skycellasn since locally I was getting cases where this failed because the match files didn't exist.  I decided that this was because something wasn't guaranteeing that run_patchlist would happen first, and adding this made this issue go away.  Does this change look sane to you?